test/serializer/abstract/ConditionalAbstractObjectValueSet.js

```diff
@@ -0,0 +1,13 @@
+// does not contain:unnecessaryIndirection
```

Comment:
This is what is different with this diff. The intermediate objects are now eliminated.